docs/docs/developers/development/dialogs/index.md

```diff
@@ -84,6 +121,65 @@ Front End validation is done by Atlaskit by default, if the submitted value is o
 WebSight supports the validation of dialog values on BackEnd side. If the value is incorrect, then it won’t be saved and the dialog can’t be submitted. The author will see a proper error message.
 ![](dialog-backend-validation.png)
 
+#### Custom validator
+To prepare a custom validator you have to extend an `AbstractValidator` form `websight-dialogs-service` as an OSGi `@Component(service = DialogValidator.class)`. 
```

Comment:
this is invalid. AbstractValidator is our impl. Custm validators must just implement pl.ds.websight.dialog.spi.DialogValidator provided by pl.ds.websight:websight-dialogs-service